share/spice/go_watch_it/go_watch_it.js

```diff
@@ -0,0 +1,102 @@
+(function (env) {
+    "use strict";
+
+    function getAlertAvailability(watchable) {
+        var avail = {
+            provider_format_logos: {
+                dark: 'http://s3.amazonaws.com/gowatchit/partner_logos/gowatchit_logo_white_bars.jpg'
+            },
+            buy_line: watchable.title,
+            format_line: "Set Alerts",
+            watch_now_url: getUrl(watchable)
+        }
+        return avail;
+    }
+
+    function getUrl(watchable) {
+        return 'http://gowatchit.com' + (watchable.url ? watchable.url : "");
+    }
+
+    env.ddg_spice_go_watch_it = function(data) {
```

Comment:
Nitpicking, but we usually go with the variable name `api_result` instead of `data` :)